hooks/lib/ecr-registry-provider.bash

```diff
@@ -1,5 +1,18 @@
 login() {
-  $(aws ecr get-login --no-include-email)
+  local aws_cli_version=$(aws --version 2>&1 | cut -d " " -f1 | cut -d "/" -f2)
+
+  if [[ $aws_cli_version =~ ^1 ]]; then
+    $(aws ecr get-login --no-include-email)
```

Comment:
AWS CLI V1 also supports `get-login-password` and has `get-login` marked as deprecated.

I'd be keen for us to drop this entirely; we can do a major version increment if need be. I'm not sure which V1 version added `get-login-password` but I think it's been around for years at this point.

Reply:
Ah, that simplifies it a lot in that case. Will push an update shortly